scripts/validate-upgrade-ssv-network.ts

```diff
@@ -1,6 +1,8 @@
+import { ethers, upgrades } from 'hardhat';
+
 async function validateUpgradeSSVNetwork() {
-  const proxyAddress = process.env.SSVNETWORK_PROXY_ADDRESS;
-  const SSVNetwork = await ethers.getContractFactory("SSVNetwork_V2");
+  const proxyAddress: any = process.env.SSVNETWORK_PROXY_ADDRESS;
```

Comment:
Not huge fan of anys, if you must well keep it but lets try to always make a type if needed (no ts-ignore either)